### DISCUSSION TITLE
Not able to reproduce YourTTS model performance

### DISCUSSION THREAD

**Discussion by pranavpawar3** [coqui-ai/TTS#1759] [General Q&A] [answered]
Hi Team,

We have been experimenting with YourTTS for quite some time and found that we couldn't reproduce the results same as that on the coqui.ai YourTTS demo on the same speaker files.

We are using this model - `https://coqui.gateway.scarf.sh/v0.6.1_models/tts_models--multilingual--multi-dataset--your_tts.zip`

This released model synthesizes the audio at a 16000 Hz sampling rate, while the model hosted on coqui.ai web API synthesizes it at 44100 Hz and with better quality.

I wanted to ask if any other pieces are required to achieve higher quality sampling? Or can we change the config parameters to achieve higher quality synthesis?

Thanks

**Comment by erogol** [comment, marked as answer]
The released model only accompanies the paper. For the one on the website, we don't have a plan to release a recipe any time soon.

**Reply by Malfaro43** [reply]
@erogol Are there still no plans to release the model?

**Reply by erogol** [reply]
beyond ```tts_models/multilingual/multi-dataset/your_tts``` we don't have plans to release a new model.
